dashboard/app/views/pd/workshop_mailer/facilitator_post_workshop.html.haml

```diff
@@ -34,7 +34,10 @@ As a reminder, there are a number of ways for you to get support between worksho
   %li
     Engage with other facilitators in the
     = link_to('Facilitator Forum', 'https://forum.code.org/c/facilitators') + ' and'
-    = link_to('Facilitator Slack channel', 'https://codefacilitators6-12.slack.com/messages')
+    - if @workshop.course == 'CS Fundamentals'
+      = link_to('Facilitator Slack channel', 'https://csffacilitators.slack.com/messages')
+    - else
+      = link_to('Facilitator Slack channel', 'https://codefacilitators6-12.slack.com/messages')
```

Comment:
@dmcavoy Did the url change for the Facilitator Slack channel or are there different urls between CSF and non-CSF? Just wondering if this if/else is needed or if we can do a single link?

Reply:
There is a K-5 facilitator and 6-12 facilitator slack channel. So there are 2